From 751f70014cea52c56a74d132000afa25109685e9 Mon Sep 17 00:00:00 2001
From: Scott Wood <scottwood@freescale.com>
Date: Tue, 4 Dec 2012 10:42:17 -0600
Subject: [PATCH 202/227] powerpc:/fsl-booke-32: fix tlb_lock build break

tlb_lock and tlb_unlock were defined for 64-bit fsl booke, but one of the functions it was
used in is shared with 32-bit, so we need no-op stubs for 32-bit.

Signed-off-by: Scott Wood <scottwood at freescale.com>
Signed-off-by: Andy Fleming <afleming@freescale.com>
[Kevin: Original patch taken from fsl sdk 1.3.1
QorIQ-SDK-V1.3.1-SOURCE-20121220-yocto.iso.]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/powerpc/mm/tlb_nohash_low.S |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/powerpc/mm/tlb_nohash_low.S b/arch/powerpc/mm/tlb_nohash_low.S
index 39783ab..4d808ba 100644
--- a/arch/powerpc/mm/tlb_nohash_low.S
+++ b/arch/powerpc/mm/tlb_nohash_low.S
@@ -254,6 +254,12 @@ END_FTR_SECTION_IFSET(CPU_FTR_476_DD2)
  * to have the larger code path before the _SECTION_ELSE
  */
 
+.macro tlb_lock
+.endm
+
+.macro tlb_unlock
+.endm
+
 /*
  * Flush MMU TLB on the local processor
  */
-- 
1.7.9.7

